documentation/modules/adding-source-provider.adoc

```diff
@@ -121,14 +158,54 @@ ifdef::rhv[]
 include::snip-certificate-options.adoc[]
 endif::[]
 ifdef::ova[]
-. Click *Open Virtual Appliance (OVA)*.
+. Access the *Create provider* page for Open Virtual Appliance by doing one of the following:
+
+.. In the {ocp} web console, click *Migration for Virtualization > Providers*.
+... Click *Create Provider*.
+... Select a *Project* from the list. The default project shown depends on the active project of {project-short}. 
++
+If the active project is *All projects*, then the default project is `openshift-mtv`. Otherwise, the default project is the sane as the active project.
++
+If you have Administrator privileges, you can see all projects, otherwise, you can see only the projects you are authorized to work with.
+... Click *Open Virtual Appliance*.
+
+.. If you have Administrator privileges, in the {ocp} web console, click *Migration for Virtualization > Overview*.
+... In the *Welcome* pane, click *Open Virtual Appliance*.
++
+If the *Welcome* pane is not visible, click *Show the welcome card* in the upper-right corner of the page, and click *Open Virtual Appliance* when the *Welcome* pane opens.
+... Select a *Project* from the list. The default project shown depends on the active project of {project-short}. 
++
+If the active project is *All projects*, then the default project is `openshift-mtv`. Otherwise, the default project is the sane as the active project.
++
+If you have Administrator privileges, you can see all projects, otherwise, you can see only the projects you are authorized to work with.
+
 . Specify the following fields:
 
 * *Provider resource name*: Name of the source provider
 * *URL*: URL of the NFS file share that serves the OVA
 endif::[]
 ifdef::ostack[]
-. Click *OpenStack*.
+. Access the *Create provider* page for {osp} by doing one of the following:
+
+.. In the {ocp} web console, click *Migration for Virtualization > Providers*.
+... Click *Create Provider*.
+... Select a *Project* from the list. The default project shown depends on the active project of {project-short}. 
++
+If the active project is *All projects*, then the default project is `openshift-mtv`. Otherwise, the default project is the sane as the active project.
```

Comment:
```suggestion
If the active project is *All projects*, then the default project is `openshift-mtv`. Otherwise, the default project is the same as the active project.
```